other_procore_apis/bim_web_viewer.md

```diff
@@ -839,6 +796,540 @@ undefined
 
 Events
 
+---
+
+## Event Names
+
+<p class="heading-link-container"><a class="heading-link" href="#event-names"></a></p>
+
+The following event names are available to notify third party code of changes in the viewer.
+
+#### Complete Example
+
+```js
+const viewer = new ProcoreBim.Webviewer(options);
+
+viewer.events.addEventListener('appResize', (data) => {
+  // Called when the window resizes and returns an object with properties
+  // `offsetHeight`, `offsetWidth`, `offsetLeft`, `offsetTop`.
+  console.log(data.offsetHeight);
+});
+
+viewer.events.addEventListener('bcfCameraSet', () => {
+  // Called when the camera has been set, does not return any data.
+});
+
+viewer.start();
+```
+
+### Callback function
+
+<p class="heading-link-container"><a class="heading-link" href="#callback-function"></a></p>
+
+The second parameter to `addEventListener` is a third party defined function to be executed when the event fires. Depending on the event, the callback function may return undefined, or data. This data may be a number, string, or a JavaScript object.
```

Comment:
Feels like this callback function was meant to go somewhere else? Maybe part of `addEventListener`

![image](https://user-images.githubusercontent.com/2865469/186730614-9e291908-44e6-489d-b9b4-5dde30b9ed07.png)